rasa/importers/skill.py

```diff
@@ -0,0 +1,188 @@
+import logging
```

Comment:
I messed up the renaming here, sorry! I removed all the static stuff and adapted it to the `TrainingFileImporter` interface.